src/components/Book.js

```diff
@@ -1,18 +1,29 @@
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { removeBookAction } from '../redux/actions';
 
-const Book = ({ book }) => {
+const Book = ({ book, removeBook }) => {
+  const handleRemoveBook = book => removeBook(book);
```

Comment:
* [ ] Move this to the booklist rather, based on the recommendations the Book component will rely on only the book props and remove the book from passed from the book list.

Reply:
Done